### ISSUE TITLE
Block item Ids aren't copied properly

### ISSUE THREAD

**Issue by ramseyspace** [mircokroon/minecraft-world-downloader#147]
I am using this to back up sky factory 4 server world to save to single player, It only copied vanilla items.

![Screenshot 2021-04-06 233637](https://user-images.githubusercontent.com/61401431/113806791-f9cb3280-9730-11eb-9f17-a4022343cd2d.png)

![Screenshot 2021-04-06 233550](https://user-images.githubusercontent.com/61401431/113806806-018ad700-9731-11eb-88e3-dee4f0c1fef6.png)

Please fix this

**Comment by mircokroon** [comment]
I guess this is a duplicate of #144 ?

**Comment by ramseyspace** [comment]
Yeah, well has saying his is fabric but this one was forge mod pack.

**Comment by mircokroon** [comment]
Ah, I see. That one is a similar issue yeah but the solution is very different since, it worked quite differently in 1.12. 

This one is straightforward to fix, just need to add some handling for the Forge packet that registers the modded blocks (same for the Fabric issue although I'm not yet sure how it works there). Hopefully I'll get around to it soon.
